pre-course1,exercise-3.java

```diff
@@ -0,0 +1,97 @@
+// Time Complexity : adding one element is o(1),but printing an element is o(n) because we have to iterate all elements.
+// Space Complexity :12+12 for three integers variable.
+// Did this code successfully run on Leetcode :no got errors.
+// Any problem you faced while coding this :yes regarding cannot find variable.
+
```

Comment:
Don't complicate ur thinking, if u have head why do you need list as an argument?